src/components/VideoPlayer.vue

```diff
@@ -458,6 +519,17 @@ export default {
                 }
 
                 const config = {
+                    controlPanelElements: [
+                        "play_pause",
+                        "time_and_duration",
+                        "spacer",
+                        "skip_segment",
```

Comment:
When SponsorBlock is not enabled, or I use the auto skip mode, wouldn't this be unnecessary?

Reply:
The button is hidden by default, it's only visible when we're in a segment, so I didn't feel that gating it further was necessary.